doc/source/reference.rst

```diff
@@ -0,0 +1,218 @@
+.. _reference:
+
+=========
+Reference
+=========
+
+This chapter contains all the extra reference information (i.e. in more
+details) about the socok8s repository.
+
+For information on how to deploy socok8s, refer to :ref:`deploymentguide`.
+
+For information on how to manage and operate socok8s, refer to
+:ref:`operationsdocumentation`.
+
+For information on how to contribute to socok8s, refer to
+:ref:`developerdocumentation`.
+
+Project history
+===============
+
+This project started as a way to build and test the OpenStack-Helm charts for
+SUSE, on our SUSE products: The Container as a Service Platform (CaaSP), and
+the SUSE Enterprise Storage (SES).
+
+It started as a series of shell scripts and ansible playbooks, choosing the
+simplest and fastest way to bring a test infrastructure for the upstream
+charts.
+
+However, the mechanism of deployment was very flexible from day one, to allow
+developers to test independently their changes. It would allow them to, like any
+user/customer would, override specific parts of the deployment.
+
+Project goals
+=============
+
+* Simplicity
+* Be stable
+* Use latest stable products from SUSE
+* Carry the strict minimum amount of code to support upstream work on SUSE products.
+* Be packagable/installable offline
+* Leverage upstream first
+
+Design considerations
+=====================
+
+In order to not pollute the developer/CI machine (called `localhost`),
+all the data relevant for a deployment (like any eventual override) is stored
+in a user-space "WORKDIR" folder, with unprivileged access.
+
+This also helps the story of running behind
+a corporate firewall: the `localhost` can be (connecting to)
+a bastion host with the "deployment" actions happening behind the firewall.
+
+Why...
+======
+
+... Ansible?
+   Using Ansible is more robust than having written socok8s fully on shell
+   scripts. It's ecosystem allows a nice interface to track deployment
+   progress with ARA, run in a CI/CD like Zuul or Tower/AWX.
+
+   On top of it, with config_template, Ansible provides a hash merge
+   mechanism which is flexible enough to have all the overrides we need.
+
+... OpenStack on top of Kubernetes on top of OpenStack by default?
+   We do have a cloud for our Engineers, and that cloud is used for CI.
+   From that point, creating a node for testing is simple as doing an API call,
```

Comment:
I believe it is _as simple as_ (same next part of the sentence). But I could be wrong of course

Reply:
Both sound fine to me, I will adapt with your comment.